portal-api/src/constants/login-results.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
-  userNotFound: 'user not found',
+  userOrPasswordIncorrect: 'username or password incorrect',
```

Comment:
A bit picky from me, but I prefer `usernameOrPasswordIsIncorrect: 'username or password is incorrect'` to match the ones below